src/components/EnvelopeList.vue

```diff
@@ -674,4 +568,7 @@ div {
 		display: block;
 	}
 }
+.action-buttons {
```

Comment:
this is a child element of `.multiselect-header`. Move it in there to get into a BEM-like structure?